invenio_rdm_migrator/streams/runner.py

```diff
@@ -59,6 +59,11 @@ def __init__(self, stream_definitions, config_filepath):
         }
         GLOBAL.STATE = StateEntity(self.state, "global", "key")
 
+        # set up secret keys
+        GLOBAL.STATE.add("old_secret_key", {"value": config.get("old_secret_key")})
```

Comment:
I'm torn between adding them to the global state because it's a security concern and they would be stored in a sqllite file on a server. An alternative would be to load them with `os.environ`, this means they will be stored in some script that sets the environment (as we do for the services' credentials).

I think no matter which option we choose we would have to store them in a file in the server, so there is no escapipng that. Storing it in the state db means that subsequent runs will have it, and in case something goes wrong (e.g. need to re-encrypt, we have the previous key stored in a db).